general/deploy.sh

```diff
@@ -34,14 +34,12 @@ if [ -d "./$SERVICE" ]; then
     echo "========================================"
     echo "SSH Preparation"
     
-    echo "$SSH_PRIVATE_KEY" >> ./id_rsa
-    echo "$SSH_PUBLIC_KEY" >> ./id_rsa.pub
-    chmod 700 id_rsa
-    chmod 700 id_rsa.pub
-    
-    mkdir -p ~/.ssh/
-    # TODO: IMPLEMENT THIS https://serverfault.com/questions/132970/can-i-automatically-add-a-new-host-to-known-hosts/807363#807363
-    ssh-keyscan -H $SERVER_HOST >> ~/.ssh/known_hosts
+    mkdir -p ~/.ssh && chmod 700 ~/.ssh
+    echo "$SSH_PRIVATE_KEY" >> ~/.ssh/id_rsa && echo "$SSH_PUBLIC_KEY" >> ~/.ssh/id_rsa.pub
```

Comment:
Untuk yang ini kayanya malah lebih bagus dibikin untuk overwrite.

```suggestion
    echo "$SSH_PRIVATE_KEY" > ~/.ssh/id_rsa && echo "$SSH_PUBLIC_KEY" > ~/.ssh/id_rsa.pub
```

Kalo ke append gini nanti keynya malah jadi gak valid kalau jalan 2 kali